app/views/projects/_project_info.slim

```diff
@@ -0,0 +1,44 @@
+ruby:
+  project = local_assigns.fetch(:project)
+
+table.table.table-striped.table-bordered.qa-project-info
+  tr
+    th Mentor
+    td
+      = link_to project.mentor_name, project.mentor rescue project.mentor_name
```

Comment:
I am not familiar with `rescue` in the view. Is it rescue-ing the link? 🤔 
(And if so, wouldn't it be needed in the next link on line 10 as well?

Reply:
Hm, interesting question! I tbh have no idea what this is doing even though I may have written it myself some time ago...

Reply:
My guess is that `link_to` buckles when `project.mentor` is `nil`. In that case it only displays the name w/o a link?